clang/test/SemaSYCL/no-prototype-func.c

```diff
@@ -0,0 +1,3 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -verify -fms-extensions %s
```

Comment:
You need at least one triple on the command-line.   Most appropriate to test would be -triple spir64